scala/scala.bzl

```diff
@@ -369,3 +550,68 @@ def scala_repositories():
     url = "https://oss.sonatype.org/content/groups/public/org/scalatest/scalatest_2.11/2.2.6/scalatest_2.11-2.2.6.jar",
     sha256 = "f198967436a5e7a69cfd182902adcfbcb9f2e41b349e1a5c8881a2407f615962",
   )
+
+def scala_2_10_repositories():
```

Comment:
this is tricky. We probably want to think about how to support different versions of scala. I assume this zinc is only working for 2.10 now?